pyangbind/plugin/pybind.py

```diff
@@ -1622,3 +1630,4 @@ def get_element(ctx, fd, element, module, parent, path,
 
     this_object.append(elemdict)
   return this_object
+
```

Comment:
Could you remove this blank line? Aiming not to make whitespace changes far from the original changes.